README.md

```diff
@@ -16,24 +16,37 @@ There three categories of metrics that Moneo monitors:
     - Power
     - ECC Counts
     - GPU Throttling
-2.	GPU Profiling Counters
+1.	GPU Profiling Counters
```

Comment:
Why is this rendered as 2. ?

Reply:
The magic of markdown.

Reply:
Its something ionherent in the markdown language

Reply:
Should be left as is then?

Reply:
> Should be left as is then?

It could have but I changed it for uniformity

Reply:
Does it get formatted differently if you number things from 1 to 3? This way it might get confusing if people view the text directly in a text editor...